JesusFilm/core
# feat: eng 849 language name in query

# Description

### Issue

When a user types a language name and then selects a language filter or suggestion with that language, we want to strip the language in question out of the query as best we can.

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-849/language-name-in-query)

### Solution

Normalise the name of the language being refined i.e. `Spanish, Latin American` -> `spanish`
Remove any occurences of this normalised language from the query. i.e. `Jesus Chinese` -> `Jesus` after `Chinese, Traditional` is selected

# External Changes

Updated the strategies view test to match pattern established

# How To Test

- If a user types jesus spanish, then selects the Spanish, Latin American language, we should set the query string to jesus
- If a user types Jesus Chinese, (with any capitals) then selects the Chinese, Traditional language, we should set the query string to Jesus
- If a user types russian, then selects the Russian, Sign Language language, we should set the query string to empty
